src/platform/backends/qemu/qemu_virtual_machine.cpp

```diff
@@ -232,7 +255,9 @@ void mp::QemuVirtualMachine::start()
         mpl::log(mpl::Level::info, vm_name, fmt::format("Resuming from a suspended state"));
 
         update_shutdown_status = true;
-        delete_memory_snapshot = true;
+        is_starting_from_suspend = true;
+        current_resume_start_time = std::chrono::steady_clock::now();
+        network_reset_wait_time = 5s;
```

Comment:
This could be a bit simpler with a `network_deadline = std::chrono::steady_clock::now() + 5s;` and then `+30s` below, no?

Reply:
I think what you added is better and should work as intended.  :+1: